.github/workflows/charterafrica-deploy-dev.yml

```diff
@@ -2,7 +2,9 @@ name: charterAFRICA | Deploy | DEV
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - chore/build-arm-image
```

Comment:
Looks good to me @koechkevin. I'm just not sure why we need to maintain the `chore/build-arm-image` branch as a workflow trigger

Reply:
No @thepsalmist this is used to test this PR I will remove it when this PR is about to merge.